projects/encouraging-engineers-to-take-action.md

```diff
@@ -1,93 +1,213 @@
 ---
-
 title: Encouraging engineers to take action
 description: We gathered our FinOps experts to address the challenge of encouraging and enabling engineers to get more involved. It's not just a challenge for engineers-- everyone has a role to play.
 layout: default
-
 ---
 
 ## Playbook Details
 
 - Status: Graduated
 - Slack room: https://finopsfoundation.slack.com/archives/G01NQLND9HR
-- SIG Ownership: 
-- Chairperson: 
+- SIG Ownership:
+- Chairperson:
 - TAC Liaison: Riley Jenkins
 
 # Encouraging engineers to take action
 
 How finance, operations, and executives can work better together at encouraging more cloud cost optimization from their engineering teams.
 
 ## Before you begin
+
 You should understand the basics of how cloud computing works, know the key services on your cloud providers, including their common use cases, and have a basic understanding of billing and pricing models. Being able to describe the basic value proposition of running in the cloud and understand the core concept of using a pay-as-you-go consumption model are also necessary.
 
 You’ll also need to have a base level of knowledge of at least one of the three main public cloud providers (AWS, Azure, Google Cloud). For AWS, we recommend AWS Business Professional training or, even better, the AWS Cloud Practitioner certification. For Google, check out the Google Cloud Platform Fundamentals course. For Azure, try the Azure Fundamentals learning path. Each can usually be completed in a full day workshop.
 
 ## Relevant FinOps Framework components
```

Comment:
@tracom1 We're going to need to re-do this once the Framework gets to where it needs to be. I think these will still resolve properly with the way things are set up today, but just heads up that there's going to be rework here (and in other Projects).

Reply:
That's future Tracy and Andrew's problem!